scripts/updateContributionsAuthors.sh

```diff
@@ -35,10 +35,16 @@ for file in $(find src/ -type f \( -name "*.hpp" \)); do
 	cat /tmp/tmp.txt > $file
 
 
-	header="/// © $projectBeginYear-$projectEndYear Hub, All Rights Reserved
-/// @author $authorName <$authorEmail>
-/// @date $authorDate
-	"
+	header="/**
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * Copyright © $projectBeginYear-$projectEndYear Hub. All Rights Reserved.
+ * @author $authorName <$authorEmail>
```

Comment:
So the script does not support files authored by several authors ?

Reply:
Author seems to be the creator of the file with the date of creation

Reply:
Do we expect to run this script regularly, or is it a bootstrap for new files ?

Reply:
We can talk about tomorrow, this script can be launch by bot CI, because we have to update all headers each year.
Should we need to add the github url for the project ?

Reply:
Ok, let discuss this AFK.